### PR TITLE
Clean up english

### PR DESCRIPTION
**Pull request by steveno**

### CODE REVIEW

**Comment by steveno** [comment]
This replaces the previous pull request. Sorry about that.

**Comment by masatake** [comment]
@vhda, can I ask you to review this?
I'm a really bad person for this task.

**Comment by vhda** [comment]
@masatake do you have any comments regarding the width of the text? Documentation has been mostly done by you, so I'm not sure if you followed any specific guideline.

File: docs/building.rst [hunk 1]
```diff
@@ -10,17 +10,17 @@ Building ctags
...
-As on some systems (e.g. BSD) there is a 'ctags' program in the base
-system it's somewhat inconvenient to have the same name for universal-ctags
-During ``configure`` you can now change the output executable name.
+On some systems, like certain BSDs, there is already a 'ctags' program in the base
+system, so it is somewhat inconvenient to have the same name for 
+universal-ctags. During the ``configure`` stage you can now change 
+the output executable name.
 
-Add a prefix 'ex' which will result in 'ctags' transformed into 'exctags'
+To add a prefix 'ex' which will result in 'ctags' being transformed into 'exctags'
```
Comment (vhda) [inline docs/building.rst]: I think the "which" would benefit from a preceding comma. Typically, when no comma is used the word "that" is used instead.

**Comment by vhda** [comment]
@steveno take into account that you can use `git push -f` to update these commits instead of creating a new Pull Request.

**Comment by steveno** [comment]
@vhda Yeah, I know. I messed up my repo and frankly it was easier to start from scratch. I *sort of* love git most days but other days I can understand why OpenBSD still uses CVS.

**Comment by vhda** [comment]
@steveno if you say that is because you don't know about `git reflog`. After you know this command it is impossible to mess your repository.
And if you do mess with your repository, then I'm sure you are doing things that are impossible to do in CVS.

**Comment by masatake** [comment]
@vhda, what I can say is LGTM. "To add" is obviously better than "Add". 

@steveno, do you read whole the documents?   I have had only a few feed-backs about the docs/*.rst.
I really want to get comments.

When releasing, we have to move some parts of the documents to ctags.1.in.

**Comment by vhda** [comment]
Ok. In that case from my part everything is LGTM.

**Comment by masatake** [comment]
@steveno, if you really want not to use git and have interests improving document, I will reopen #353 :)

We really need volunteers about this area; I have been written some documents but they may be broken. It prevents releasing.

**Comment by masatake** [comment]
Thank you for the contribution.

**Comment by steveno** [comment]
Sorry about the delay! Hectic week!

@masatake I did not read *all* of the docs. I wouldd be happy to review some specific ones if you want. Just let me know.

@vhda I will have to checkout `git reflog`

**Comment by masatake** [comment]
currently we have two docuemts.

1. ctags.1: These are for (A) normal user. Stable feature should be written here.
2. docs/*.rst: These are for (A) people who want to hack ctags, and (B) staging for ctags.1.

One of thing I expect to doc reviewers is comments about which new features we wrote in *.rst can be moved to ctags.1. 

I got comments about optlib. Its directory structure is too complex. This kind of comments helps me to know what kind of feature I should work on more.

In other word, I would like to ask people which new feature do you like?
If you have a topic you would like to know, tell me.

**Pull request merged**